cmake/Dependencies.cmake

```diff
@@ -1,9 +1,9 @@
 TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
   SUBPACKAGES_DIRS_CLASSIFICATIONS_OPTREQS
     Utils                 packages/Utils                      ST  REQUIRED
-    Interface             packages/Interface                  ST  REQUIRED
+    Interface             packages/Interface                  ST  OPTIONAL
     Search                packages/Search                     ST  REQUIRED
-    Discretization        packages/Discretization             ST  REQUIRED
-    Meshfree              packages/Meshfree                   ST  REQUIRED
+    Discretization        packages/Discretization             ST  OPTIONAL
+    Meshfree              packages/Meshfree                   ST  OPTIONAL
```

Comment:
This feels very arbitrary to have only `Search` and `Utils` as required packages. What can you do with the `Search` alone? I think that `Interface` and `Meshfree` should be required.

Reply:
It takes for ever to build on my laptop and I cannot disable if it is required. Anyway I am not planning on committing these changes to maste, this is a work in progress.